applications/crossbar/src/modules/cb_clicktocall.erl

```diff
@@ -460,7 +460,7 @@ build_originate_req(Contact, Context) ->
        ,{<<"Outbound-Caller-ID-Number">>, Caller#contact.number}
        ,{<<"Outbound-Call-ID">>, CallId}
        ,{<<"Ringback">>, wh_json:get_value(<<"ringback">>, JObj)}
-       ,{<<"Dial-Endpoint-Method">>, <<"single">>}
```

Comment:
If we are requesting `loopback` endpoint, a responder will determinate `dial-method`